tests/components/destination_list.spec.js

```diff
@@ -90,7 +90,7 @@ describe("DestinationList", () => {
     const onClick = jest.fn();
     const component = shallow(<DestinationList unselectItems={onClick} />);
     const list = component.find(List).at(0);
-    list.simulate("click", {});
-    expect(onClick).toHaveBeenCalledWith([{}]);
+    list.simulate("click", {}, { id: {} });
```

Comment:
We expect to see unselectedItems([id])
not unselectedItems([{id: {}}]) 
no?

Reply:
I don't think so, it took all the second param and wrap it with array

Reply:
So why is it id and not item at destination_list.js